src/Compiler/Checking/Expressions/CheckExpressions.fsi

```diff
@@ -351,6 +351,12 @@ type TcCanFail =
     | IgnoreAllErrors
     | ReportAllErrors
 
+[<RequireQualifiedAccess>]
+[<Struct>]
+type TcTrueMatchClause =
```

Comment:
Maybe give this a short `///` description? So it's quicker for readers to understand what true match stands for.

Reply:
Done